templates/page.tmpl

```diff
@@ -116,11 +144,8 @@
         </section>
     </footer>
 
-    <!-- dark/light support -->
-    <ul class="mode-switch">
-        <li class="mode-switch-icon"></li>
-        <li><label class="switch"><input type="checkbox"><span class="slider round"></span></label></li>
-    </ul>
+    <script defer src="<TMPL_VAR BASEURL>script/navscroll.js"></script>
+        <script defer src="<TMPL_VAR BASEURL>script/mobile.js"></script>
     <script defer src="<TMPL_VAR BASEURL>script/dark-light-mode-switch.js"></script>
```

Comment:
As fas as I can see, these do not need separate encapsulation and call all be combined into a single script. The fact that they are separate files just adds overhead.